python/ql/test/experimental/query-tests/Security-new-dataflow/CWE-502/options

```diff
@@ -0,0 +1 @@
+semmle-extractor-options: --max-import-depth=2 -p ../lib
```

Comment:
Are you sure you need this now? (I would bet that you don't)

```suggestion
```

Reply:
I just copied it from the old test. If it is not needed, I should delete the whole file.

Reply:
Gone